### ISSUE TITLE
[IMPROVE] Current AE2 Compatibility

### ISSUE THREAD

**Issue by Reginald-Halifrax** [DarkholmeTenk/TardisModIssues#248]
I'm writing on github as you had asked.

First of all can't thank you enough for the fluid interface fix. no issues anymore.

But with the abbility of fluid being fixed I found more of a.. "Irritation"...

The Tardis ME roundels can only handle 32 networks at a time... that is extremely restrictive, and pretty much throws out the point of ME Roundels.

So my request is too make roundels able to More or less hold a infinite amount of networks (or just a high number no one can reach) but do not have the roundels able to directly connect to a controller.

its a sorta "additive" network grid. the more dense cables you have going out of the computer into the roundels the more space you can work from. making it so you do still have to work for a "large" network.

id really rather not (nor would anyone else i would bet) would have to make a P2P system based on sectors because even then you will eventually get restricted....

TL;DR The ME interface is just this: 0/32 networks used. for each dense cable put from the controller to a roundel (or regular cable) would make it 0/40 networks or 0/64 networks.

heck if you felt like it you could have a network buffer as a sortof "upgrade" you could get at the temporal engine if it detects AE2 is installed. would still be balanced in my book.

**Comment by DarkholmeTenk** [comment]
That isn't how AE2 works, the only thing I can reasonably implement is an IGridHost, there's no way I can make the roundels into one big controller and I have absolutely no control over how many channels are available.

Basically how the ME roundel works is they all act as one big AE machine which can carry a dense signal through.

Unless you can find a way to implement an AE controller just by using the API or control the number of channels available, I literally can't do this.
See https://github.com/AppliedEnergistics/Applied-Energistics-2/tree/master/src/api/java/appeng/api for the API

**Comment by DarkholmeTenk** [comment]
On further inspection, I may be able to implement my own IGridConnection and do it that way. Eww.

**Comment by Reginald-Halifrax** [comment]
Well i know i couldn't be happier with a additive network grid.

i don't blame you if you don't want to try its just that this is not just "my problem" as you could guess.
others have to deal with this too if they want a grid.

I know how to use AE but i hardly know anything about its API.

**Comment by DarkholmeTenk** [comment]
On further inspection, IGridConnection is not implementable for other mods. I honestly can't see any way to extend the number of channels available.

**Comment by Reginald-Halifrax** [comment]
Well I do thank you for your time. I'll try to do a P2P system based on sectors. feel free to close this post.

**Comment by DarkholmeTenk** [comment]
If it's really inconvenient for you, I recommend disabling network usage in AE's config, or wait until the linking system is implemented full :P

**Comment by ruifung** [comment]
LINKING SYSTEM. Does this mean some kind of pairing system? Or maybe the ability to split roundels into networks?

**Comment by FoxPotato** [comment]
I think the plan will be that you can link 2 roundels, which have lets say item interfaces, so you can transport items between those two. iirc tho.

**Comment by DarkholmeTenk** [comment]
Linking 2 roundels (for AE purposes) done

**Comment by Firestar311** [comment]
@DarkholmeTenk Is there a way to allow colored cable support so that only colored cables link to the same colored cable within the tardis? So lets say you have a controller in a room with the 16 color cables running from it into roundels, is there a way that you can "Link" this with an item or something. I have seen this done in different ways in other mods.

**Comment by DarkholmeTenk** [comment]
nope, but you can link ae2 cable interfaces together in the ds build

**Comment by Firestar311** [comment]
Where do I find the DS build.

**Comment by DarkholmeTenk** [comment]
I don't think any packs have this version of the DS build, but it'll be coming out sooon :tm: anyways
